### PR TITLE
Discussion PR - Integration tests for entity group transactions

### PR DESCRIPTION
**Pull request by edwin-huber**
As discussed, we have separated out the initial integration tests for entity group transactions / batch into a separate PR to allow some parallel work.

Suggestion is that the tests are written against the service, and then refactored to Azurite.

Serialization and De-Serialization of batch requests are covered by unit tests and will be submitted with the PR for table batch.

### CODE REVIEW

File: tests/table/apis/table.entity.test.ts [hunk 1]
```diff
@@ -436,4 +447,130 @@ describe("table Entity APIs test", () => {
       }
     );
   });
+
+  it.only("Simple batch test: Inserts multiple entities as a batch, @loki", (done) => {
+    requestOverride.headers = {
+      Prefer: "return-content",
+      accept: "application/json;odata=fullmetadata"
+    };
+    const batchEntity1 = createBasicEntityForTest();
```
Comment (XiaoningLiu) [inline tests/table/apis/table.entity.test.ts:456]: Table requires unique combinations of "PartitionKey" and "RowKey" for different entities.

**Review by XiaoningLiu — APPROVED** [review]
One comment added.

We can merge this to unblock others keeping contributing to Batch test cases.

Seems pipeline failed with the change, can we make pipeline green?

**Comment by edwin-huber** [comment]
Hi,  
I submitted this as a PR for discussion, as the point of the integration tests is that they will fail until we get the implementation right...  
I see I forgot the "only" designation on the insert test, but all tests for batch will fail.  
I'm happy to collaborate with whomever is writing the tests, but we would need to invalidate the tests if we want the builds to complete, which removes the point of committing the tests until batch is ready.

Your thoughts?

**Comment by XiaoningLiu** [comment]
> writing the tests, but we would need to invalidate the tests if we want the builds to complete, which removes the point of committing the tests until batch is ready.

Ok to leave the PR here with cases failures. @tzhanl can you help pick up from here?

File: tests/table/apis/table.entity.test.ts (hunk 1 above)
Comment (edwin-huber) [reply tests/table/apis/table.entity.test.ts:456]: Thanks, yes.  
This is why the helper function creates a new row for each entity 😸 
```javascript
getUniqueName("row")
```

**Comment by edwin-huber** [comment]
Now implemented and merged in table branch.

**Pull request closed without merging**